src/main/java/lib/controllers/DieterController.java

```diff
@@ -28,13 +31,20 @@ public DieterController(double kp, double ki, double kd, double kDieter, double
         this.kDieter = kDieter;
     }
 
+    public final int x = 0;
+
     public DieterController(double kp, double ki, double kd, double kDieter) {
         super(kp, ki, kd);
         this.kDieter = kDieter;
+        Stream.of("1", 2);
+        int i = 2;
+        i = i + 3;
+        System.out.println(i > 2 ? i : i + 1);
```

Comment:
I added that while testing the CleanThat refactorer and forgot to delete it, sorry about that. I don't want to delete it myself though to prevent merge conflicts, so suggesting:
```suggestion
```

Reply:
6cc9252e8b95fa8bd4879bd3ec7ec07df42ea5e6